app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -2018,7 +2018,13 @@ class BrowserTabFragment :
     }
 
     fun onLongPressBackButton() {
-        viewModel.onUserLongPressedBack()
+        /*
+         It is possible that this can be invoked before Fragment is attached
+         If viewModelFactory isn't initialized, ignore long press
+         */
+        if (this::viewModelFactory.isInitialized) {
+            viewModel.onUserLongPressedBack()
+        }
```

Comment:
I think this is theoretically possible so worth a shot. I wonder why the initialisation would be that slow that the user would have time to click overflow menu and long press, that's prob seconds.

But, on the other side, this fragment has a ton of dependencies, so DoubleChecks on them can cause contingency

Reply:
It isn't from the overflow; it's long press on the system back button (both options are possible, but I know from the stack trace it's the system back button). So that is more easily accessible.

But also thinking this might be exacerbated when the main thread is getting hammered; perhaps the user has hit the back button and it's being interpreted as a long press (i've seen this happen sometimes under heavy main thread load).

I can replicate the crash by adding a synthetic delay before calling `commit` on the fragment transaction, and long pressing in this window.

It's not the most solid of theories, but might explain why it only affects small number of users since it needs a few things going wrong at the same time to happen. But, even if the theory is wrong, it's a safe enough change (albeit, ugly)